doc/Support/FAQ.md

```diff
@@ -28,6 +28,7 @@ source: Support/FAQ.md
  - [What is the Difference between Disable Device and Ignore a Device?](#faq28)
  - [Why can't Normal and Global View users see Oxidized?](#faq29)
  - [What is the Demo User for?](#faq30)
+ - [Why does modifying 'Default Alert Template' fail?](#faq31)
```

Comment:
I'd say it needs to be why does the default alert template not appear in the templates list.

If it's not in the DB you won't see it to even modify it.